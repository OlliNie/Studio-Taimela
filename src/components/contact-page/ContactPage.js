import React from 'react';
import styles from './ContactPage.css';
import { aboutVirpi } from '../../text/contact';
import Title from '../title/Title';
import ContactOptions from './ContactOptions';
import RenderParagraph from '../render-paragraph/RenderPagragraph';

export default function ContactPage() {
  return (
    <section id="Contact" className={[styles.section, 'page'].join(' ')}>
      <Title title="get in contact" />
      <ContactOptions />
      <RenderParagraph content={aboutVirpi} />
    </section>
  );
}
