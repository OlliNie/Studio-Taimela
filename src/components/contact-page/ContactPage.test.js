import React from 'react';
import { shallow } from 'enzyme';
import ContactPage from './ContactPage';

describe('ContactPage component', () => {
  it('renders ContactPage', () => {
    const wrapper = shallow(<ContactPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
