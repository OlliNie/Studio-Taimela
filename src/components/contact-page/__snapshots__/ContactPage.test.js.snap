// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage component renders ContactPage 1`] = `
<section
  className="section page"
  id="Contact"
>
  <Title
    title="get in contact"
  />
  <ul
    className="ul"
  >
    <MakeContactOptions
      contactOptions={
        Array [
          Object {
            "icon": <FaEnvelopeOpenText
              size={32}
            />,
            "text": "Virpi@studiotaimela.com",
          },
          Object {
            "icon": <FaMapMarked
              size={32}
            />,
            "text": <React.Fragment>
              <span>
                Zürich
              </span>
              <span>
                Switzerland
              </span>
            </React.Fragment>,
          },
          Object {
            "icon": <FaPhone
              size={32}
            />,
            "text": <React.Fragment>
              <span>
                +41 77 979 9734
              </span>
              <span>
                 +358 40 410 6428
              </span>
            </React.Fragment>,
          },
        ]
      }
    />
  </ul>
  <p
    className="paragraph"
  >
    When I’m not at my desk, you can find me traveling or enjoying the outdoors. I’m also an
avid painter, a mother and a lifelong learner. I love discovering cultures, places and people,
and I’ve lived in five different countries with Zürich, Switzerland, my current home. 
  </p>
</section>
`;
